lib/state/simperium/middleware.ts

```diff
@@ -60,6 +92,13 @@ export const middleware: S.Middleware = store => {
       case 'REVISIONS_TOGGLE':
         fetchRevisions(store, nextState);
         break;
+
+      case 'TOGGLE_ANALYTICS_ENABLED':
+        setAnalyticsEnabled(nextState);
```

Comment:
one value of passing `doEnable` as a property on toggle-able actions is that we can reuse the same action instead of creating separate ones for on/off/toggle

```ts
case 'TOGGLE_ANALYTICS_ENABLED':
	setAnalyticsEnabled( action.doEnable ?? ! state.settings.analyticsEnabled );
	break;
```